### ISSUE TITLE
[Regression] Cannot escape Markdown sequences

### ISSUE THREAD

**Issue by Luhav** [ajbura/cinny#1474]
### Describe the problem

Prior to Cinny v3, one could escape a Markdown sequence by placing a backslash before the sequence, preventing it from being rendered as Markdown. This appears to have been removed from Cinny v3.
![animated GIF of Cinny V3 demonstrating various attempted escapes of Markdown formatting which do not work as intended](https://github.com/cinnyapp/cinny/assets/43188089/d2e37669-97b4-4157-aa2b-e0a509bb7ae4)

### Describe the solution you'd like

`\` should once again escape Markdown sequences, as in Cinny v2
![animated GIF of Cinny V2 demonstrated the above attempted escapes of Markdown formatting, and working as intended](https://github.com/cinnyapp/cinny/assets/43188089/9807b3e1-c8ca-4bcc-a264-1773deefe1fd)

### Alternatives considered

Simply toggling the Markdown on/off before each message is not a valid solution for the following reasons: 
- It requires 4 steps to enable/disable Markdown every time I want to switch
- Sometimes I also want to mix Markdown with characters I don't want parsed, such as using an `*` for a footnote (instead of making a large chunk of message italic) ![animated GIF of Cinny V3 demonstrating an attempted use of the asterisk as a footnote, wherein a large chunk of text ends up confusingly italic and breaking Markdown formatting later in the message](https://github.com/cinnyapp/cinny/assets/43188089/4fdd4599-cb52-4223-abba-8bbbdc9b1f77)

**Comment by rail01** [comment]
This issue is very annoying, currently the only way to escape MD syntax seems to be with inline-code, which isn't ideal when I just want to write something that isn't supposed to be MD but isn't supposed to be code either.

**Comment by rail01** [comment]
still an issue

**Comment by ajbura** [comment]
> still an issue

Yeah, we had some other high priority task. feel free to contribute, or we will sure fix it sooner.

**Comment by ajbura** [comment]
@rail01 i have done minimal effort to fix it at https://github.com/cinnyapp/cinny/pull/2208

feel free to test and let us know if that lead us in good direction.
